daffodil-sapi/src/main/scala/org/apache/daffodil/sapi/Daffodil.scala

```diff
@@ -86,6 +86,14 @@ object ValidationMode extends Enumeration {
   }
 }
 
+/**
+ * XMLOutputStyle determines how to wrap values of elements of type xs:string
+ */
+object XMLOutputStyle extends Enumeration {
+  type XMLOutputStyle = Value
+  val Standard, PrettyPrintSafe = Value
+}
+
```

Comment:
I concur with Steve about moving XMLTextEscapeStyle to the infoset package (one directory below), and I also want to mention that Scala 3 is deprecating scala.Enumeration in favor of enum Type { NAME1, NAME2, ... }, while Enumeratum is recommended for Scala 2.X (https://stackoverflow.com/questions/1898932/case-objects-vs-enumerations-in-scala).